week1/assignment/assignment1.css

```
@@ -0,0 +1,309 @@
+@font-face {
+    font-family: 'Material Icons';
+    font-style: normal;
+    font-weight: 400;
+    src: url(https://example.com/MaterialIcons-Regular.eot); /* For IE6-8 */
+    src: local('Material Icons'),
+      local('MaterialIcons-Regular'),
+      url(https://example.com/MaterialIcons-Regular.woff2) format('woff2'),
+      url(https://example.com/MaterialIcons-Regular.woff) format('woff'),
+      url(https://example.com/MaterialIcons-Regular.ttf) format('truetype');
+
+      font-family: 'SUIT Variable', sans-serif;
+  }
+
+body{
+    background-color: black;
+}
+
+  .material-icons {
+    font-family: 'Material Icons';
+    font-weight: normal;
+    font-style: normal;
+    font-size: 24px;  /* Preferred icon size */
+    display: inline-block;
+    line-height: 1;
+    text-transform: none;
+    letter-spacing: normal;
+    word-wrap: normal;
+    white-space: nowrap;
+    direction: ltr;
+  
+    /* Support for all WebKit browsers. */
+    -webkit-font-smoothing: antialiased;
+    /* Support for Safari and Chrome. */
+    text-rendering: optimizeLegibility;
+  
+    /* Support for Firefox. */
+    -moz-osx-font-smoothing: grayscale;
+  
+    /* Support for IE. */
+    font-feature-settings: 'liga';
+  }
+
+
+.header{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: white;
+    position :fixed; 
+    z-index: 999;
+    width: 100%;
+    margin-top:30px;
+}
+
+.logo{
+    width:100px;
+    padding-right: 20px;
+}
+
+.left{
+    display: flex;
+    flex-direction: row;
+    margin-left: 100px;
+}
+
+.right{
+    display: flex;
+    align-items: center;
+    margin-right: 100px;
+}
+
```

p2: `.left`와 `.right`의 스타일에서 각각 100px을 `margin`을 주고 있네요!!
개인적으로 코드를 딱 봤을 때 예쁘지도 않고, `margin`을 각각 주는 것도 번거롭다고 생각합니다.
이렇게 작성해주신 의도는 아무래도 header에서 양쪽으로 너무 딱 달라붙어서 여백을 주기 위해서겠죠??ㅎㅎ

굳이 이렇게 작성하기보다는 아래처럼 작성하면 더 깔끔한 코드가 될 것 같아요!

```suggestion
.left{
    display: flex;
    flex-direction: row;
}

.right{
    display: flex;
    align-items: center;
}

```
위에서 header에 padding을 주는 방식으로 바꿔주세요!
```css
.header {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  color: white;
  position: fixed;
  z-index: 999;
  width: 100%;
  box-sizing: border-box; /* padding값 포함해서 width 100%로 꽉차게 box-sizing 설정*/
  padding: 30px 100px 0; /* header에서 요소들 원하는 위치에 두기 위해 padding 설정*/
}
```